### ISSUE TITLE
Heal vs. Rest

### ISSUE THREAD

**Issue by jamesbmayr** [maximforever/bossfight#5]
For game balance, I think heal is a better option than rest (and makes dodge much more important). Heal would mean increasing the health, strength, and speed of teammates only, instead of rest, which increases those stats only for that player.

In a game that includes rest, it is always better to rest than to dodge - one can increase your stats, while the other can only slow the decrease. However, a game with heal means you can never directly increase your stats, and must depend on others to cooperate, so dodging is a viable strategy if your health is low enough that you need to be healed.

tl;dr: heal adds a little more depth of strategy.

**Comment by maximforever** [comment]
Love it. This increases collaboration amongst teammates, deepens strategy,
and makes a generic little game a bit more interesting. Make it so.

**Comment by jamesbmayr** [comment]
Cool. Actually, it's already built, it's just not accessible from the front-end. Everything is sandboxed as separate functions, so you just need to build a button named "heal" and replace the button named "rest". For now, you can test out player moves by typing into the admin console. Seriously, try the admin console!
